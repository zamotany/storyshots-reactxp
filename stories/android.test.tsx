import initStoryshots from '@storybook/addon-storyshots';
import path from 'path';

jest.mock('react-native', () => {
  const RN = require.requireActual('react-native');
  RN.AccessibilityInfo = undefined;
  return new Proxy(
    {},
    {
      get: (_obj, prop) => {
        if (prop === 'AccessibilityInfo') {
          return undefined;
        }
        return RN[prop];
      },
    },
  );
});

initStoryshots({
  framework: 'react-native',
  configPath: path.join(__dirname, '../native/storybook'),
});
