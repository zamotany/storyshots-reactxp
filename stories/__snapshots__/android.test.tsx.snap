// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button component All Variants 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-around",
    }
  }
>
  <View
    style={
      Object {
        "width": 200,
      }
    }
  >
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Click me"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      importantForAccessibility="yes"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#0078FF",
          "borderRadius": 8,
          "height": 48,
          "justifyContent": "center",
          "opacity": 1,
          "overflow": "hidden",
          "paddingHorizontal": 16,
          "paddingVertical": 8,
        }
      }
      tooltip="Click me"
    >
      <Text
        style={
          Array [
            Object {
              "includeFontPadding": false,
              "textAlignVertical": "center",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "Roboto-Regular",
                "fontSize": 18,
                "lineHeight": 27,
                "textAlign": "center",
              },
              undefined,
            ],
          ]
        }
      >
        Click me
         
        android
      </Text>
    </View>
    <View
      style={
        Object {
          "height": 10,
        }
      }
    />
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Click me"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      importantForAccessibility="yes"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#0078FF",
          "borderRadius": 8,
          "height": 48,
          "justifyContent": "center",
          "opacity": 0.2,
          "overflow": "hidden",
          "paddingHorizontal": 16,
          "paddingVertical": 8,
        }
      }
      tooltip="Click me"
    >
      <Text
        style={
          Array [
            Object {
              "includeFontPadding": false,
              "textAlignVertical": "center",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "Roboto-Regular",
                "fontSize": 18,
                "lineHeight": 27,
                "textAlign": "center",
              },
              undefined,
            ],
          ]
        }
      >
        Click me
         
        android
      </Text>
    </View>
    <View
      style={
        Object {
          "height": 10,
        }
      }
    />
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Click me"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      importantForAccessibility="yes"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#E8EDF1",
          "borderRadius": 8,
          "height": 48,
          "justifyContent": "center",
          "opacity": 1,
          "overflow": "hidden",
          "paddingHorizontal": 16,
          "paddingVertical": 8,
        }
      }
      tooltip="Click me"
    >
      <Text
        style={
          Array [
            Object {
              "includeFontPadding": false,
              "textAlignVertical": "center",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "Roboto-Regular",
                "fontSize": 18,
                "lineHeight": 27,
                "textAlign": "center",
              },
              Object {
                "color": "#252A32",
              },
            ],
          ]
        }
      >
        Click me
         
        android
      </Text>
    </View>
    <View
      style={
        Object {
          "height": 10,
        }
      }
    />
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Click me"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      importantForAccessibility="yes"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#E8EDF1",
          "borderRadius": 8,
          "height": 48,
          "justifyContent": "center",
          "opacity": 0.2,
          "overflow": "hidden",
          "paddingHorizontal": 16,
          "paddingVertical": 8,
        }
      }
      tooltip="Click me"
    >
      <Text
        style={
          Array [
            Object {
              "includeFontPadding": false,
              "textAlignVertical": "center",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "Roboto-Regular",
                "fontSize": 18,
                "lineHeight": 27,
                "textAlign": "center",
              },
              Object {
                "color": "#252A32",
              },
            ],
          ]
        }
      >
        Click me
         
        android
      </Text>
    </View>
    <View
      style={
        Object {
          "height": 10,
        }
      }
    />
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Click me"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      importantForAccessibility="yes"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#FFFFFF",
          "borderColor": "#252A32",
          "borderRadius": 8,
          "borderStyle": "solid",
          "borderWidth": 1,
          "height": 48,
          "justifyContent": "center",
          "opacity": 1,
          "overflow": "hidden",
          "paddingHorizontal": 16,
          "paddingVertical": 8,
        }
      }
      tooltip="Click me"
    >
      <Text
        style={
          Array [
            Object {
              "includeFontPadding": false,
              "textAlignVertical": "center",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "Roboto-Regular",
                "fontSize": 18,
                "lineHeight": 27,
                "textAlign": "center",
              },
              Object {
                "color": "#252A32",
              },
            ],
          ]
        }
      >
        Click me
         
        android
      </Text>
    </View>
    <View
      style={
        Object {
          "height": 10,
        }
      }
    />
    <View
      accessibilityComponentType="button"
      accessibilityLabel="Click me"
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      importantForAccessibility="yes"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#0078FF",
          "borderRadius": 8,
          "height": 48,
          "justifyContent": "center",
          "opacity": 0.2,
          "overflow": "hidden",
          "paddingHorizontal": 16,
          "paddingVertical": 8,
        }
      }
      tooltip="Click me"
    >
      <Text
        style={
          Array [
            Object {
              "includeFontPadding": false,
              "textAlignVertical": "center",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "Roboto-Regular",
                "fontSize": 18,
                "lineHeight": 27,
                "textAlign": "center",
              },
              undefined,
            ],
          ]
        }
      >
        Click me
         
        android
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Button component Test 1`] = `
<View
  accessibilityComponentType="button"
  accessibilityLabel="Click me"
  accessibilityTraits={
    Array [
      "button",
    ]
  }
  importantForAccessibility="yes"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "#E8EDF1",
      "borderRadius": 8,
      "height": 48,
      "justifyContent": "center",
      "opacity": 1,
      "overflow": "hidden",
      "paddingHorizontal": 16,
      "paddingVertical": 8,
    }
  }
  tooltip="Click me"
>
  <Text
    style={
      Array [
        Object {
          "includeFontPadding": false,
          "textAlignVertical": "center",
        },
        Array [
          Object {
            "color": "white",
            "fontFamily": "Roboto-Regular",
            "fontSize": 18,
            "lineHeight": 27,
            "textAlign": "center",
          },
          Object {
            "color": "#252A32",
          },
        ],
      ]
    }
  >
    Click me
     
    android
  </Text>
</View>
`;
